src/components/BaseNavbar.vue

```diff
@@ -1,55 +1,72 @@
 <template>
-<nav class="bg-white shadow w-full">
-    <div class="flex justify-between h-16 mx-auto px-4 sm:px-6 lg:px-8">
+  <nav class="bg-white shadow w-full">
+    <div class="flex justify-between h-16 px-6">
       <div class="flex">
-        <router-link to="/" class="flex-shrink-0 flex items-center mr-10">
-          <img class="block lg:hidden h-8 w-auto" src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg" alt="Workflow">
-          <img class="hidden lg:block h-8 w-auto" src="https://tailwindui.com/img/logos/workflow-logo-indigo-600-mark-gray-800-text.svg" alt="Workflow">
+        <router-link to="/blogs" class="flex-shrink-0 mr-8 flex items-center">
+          <img
+            class="h-8 w-auto"
+            src="https://tailwindui.com/img/logos/workflow-logo-indigo-600-mark-gray-800-text.svg"
+            alt="Workflow"
+          />
         </router-link>
-        <div class="hidden md:ml-6 md:flex md:space-x-8">
-            <router-link class="active-router-link" to="/">
-                     {{ $t('navbarHome.name') }}
-            </router-link>
-            <router-link class="router-link" to="/">
-                    {{ $t('navbarEditBlog.name') }}
-            </router-link>
-            <router-link class="router-link" to="/">
-                      {{ $t('navbarCreateBlog.name') }}
-            </router-link>
+        <div v-if="userState.loggedIn" class="hidden md:flex md:space-x-8">
+          <router-link class="router-link" to="/blogs">
+            {{ $t('general.Blogs.title') }}
```

Comment:
in order to  be consistent , replace Blog with blog
`$t('general.blogs.title')`